standard/info/wikis/clashofclans/info.lua

```diff
@@ -29,4 +29,5 @@ return {
 
 	defaultTeamLogo = 'Clash of Clans default allmode.png', ---@deprecated
 	defaultTeamLogoDark = 'Clash of Clans default allmode.png', ---@deprecated
+	match2 = 0,
```

Comment:
```suggestion
	match2 = 2,
```
although not on git ...
(new wiki without old match stuff)

Reply:
as above